src/task11/task11

```diff
@@ -0,0 +1,23 @@
+var ancestry = JSON.parse(ANCESTRY_FILE);
+
+const average = array => array.reduce((a, b) => a + b) / array.length;
+
+var groupBy = (items, func) => // It behaves strangely when I'm trying to make it =>. Of course I might be doing sth wrong
+    items.reduce(function (rv, x) {
+        (rv[func(x)] = rv[func(x)] || []).push(x);
+        return rv;
+    }, []);
+
+console.log(groupBy(ancestry, a => Math.ceil(a.died / 100)))
+
+const averageExpByCent = array =>
+    array.map(cent =>
+        array.indexOf(cent) + 'th: ' +
+        average(cent.map(person =>
+            person.died - person.born
+        ))
+    ).filter(res =>
+        res != null
+    );
+
+console.log(averageExpByCent(groupBy2(ancestry, a => Math.ceil(a.died / 100))))
```

Comment:
groupBy2?
Run your code before sending it to me

Reply:
yeah, that's a testing leftover
but overall, I have like 16 empty records as a the result of groupBy! It's not the solution they were hoping for, is it?

Reply:
AGA!
And I just noticed I always ask students to groupBy data into the object type, not array))
I must have missed that(

And I also noticed that you put `a => Math.ceil(a.died / 100)` several times, suggestion, why dont you just put it straight into the groupBy function and remove its 2nd argument once and for all))

Reply:
And rename **groupBy** according to the change

Reply:
And adopt new object data further in the code

Reply:
I haven't done it with the object because I didn't know how to design the second function, as HOFs work only with arrays:(
but if you say I am to do it with the object, I'll try :)

Reply:
> And I also noticed that you put `a => Math.ceil(a.died / 100)` several times, suggestion, why dont you just put it straight into the groupBy function and remove its 2nd argument once and for all))

Well, I had actually used it twice in the same context, to demonstrate the usage of the groupBy separately
changed it, and **now** it is used repetitively

Reply:
What I meant is to develop groupBy function as a helper functions, not a separate task.
In this way you could:
`groupBy(ancestry, person => Math.ceil(person.died / 100));`